commitizen/git.py

```diff
@@ -105,27 +105,12 @@ def get_commits(
     start: Optional[str] = None,
     end: str = "HEAD",
     *,
-    log_format: str = "%H%n%s%n%an%n%ae%n%b",
-    delimiter: str = "----------commit-delimiter----------",
     args: str = "",
 ) -> List[GitCommit]:
     """Get the commits between start and end."""
-    git_log_cmd = (
-        f"git -c log.showSignature=False log --pretty={log_format}{delimiter} {args}"
-    )
-
-    if start:
-        command = f"{git_log_cmd} {start}..{end}"
-    else:
-        command = f"{git_log_cmd} {end}"
-    c = cmd.run(command)
-    if c.return_code != 0:
-        raise GitCommandError(c.err)
-    if not c.out:
-        return []
-
+    git_log_entries = _get_log_as_str_list(start, end, args)
     git_commits = []
-    for rev_and_commit in c.out.split(f"{delimiter}\n"):
```

Comment:
I wonder if this `\n` was working properly for Windows.

Reply:
Our CI runs on Windows machine as well so I think it should be fine 🤔 But both maintainers do not have windows machine to test it out. Might need others' help to verify it.